packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -333,11 +333,7 @@ export const createMynahUI = (ideApi: any, amazonQEnabled: boolean) => {
                 return
             }
 
-            if (tabsStorage.getTab(tabID)?.type === 'featuredev') {
-                mynahUI.addChatItem(tabID, {
-                    type: ChatItemType.ANSWER_STREAM,
-                })
```

Comment:
I'm curious about how removing this fixes the issue. We still need one "Generating your answer" message - does that come from somewhere else?

Reply:
I suspect this can be the issue, since adding the empty streaming card (generating your answer preloader) usually gets done on the initial request, but as i see also on async process we're adding it. And in the general block not /transform or /dev or cwc specific. @JuliaABurch Can you please check?

Reply:
Following up - I reverted the section of code @dogusata pointed to and tested, and while it fixed the multiple `Generating your answer` messages, other issues arose in both `featuredev` and `gumby`. We have varying ways of creating blank cards, cards in progress, etc, that isn't consistent across controllers. This change instead bumps the version of `mynah-ui` to one that preventatively checks to make sure multiple blank cards aren't being pushed to chat messages.